MyCacheConfig.json

```diff
@@ -0,0 +1,25 @@
+{

```

Comment:
Just to check - this won't be checked in, right?

Reply:
Oh no, sorry. This was an unintended side-effect of "git add -A :/"

---
In reply to: [306406002](https://github.com/microsoft/BuildXL/pull/647#discussion_r306406002) [](ancestors = 306406002)